### ISSUE TITLE
Update for 20.03.18?

### ISSUE THREAD

**Issue by choi-choi** [roguelike2d/TekkenBot#68]
Overlay isn't working anymore using the current patch (22.02.2018).

They released Noctis yesterday as well. Do we still have plans for a new patch? This overlay has been very helpful.

**Comment by SpankiyTV** [comment]
I'm having same problem, stopped working after noctis update. this has been so helpful and has made me understand the game better.

**Comment by compsyguy** [comment]
I'm working on getting something working, but it probably won't be till next week at the earliest. I'm having to figure out how to get the memory addresses... I think someone did figure something out, but there are other issues with this build...

**Comment by sneaky-coyote** [comment]
@compsyguy hit me up, i want to figure this out.

https://twitter.com/sneaky_coyote

**Comment by piecedigital** [comment]
I think I figured out how to get the memory address (at least one) but I'm having trouble building it.

**Comment by FuzionDragonZKid** [comment]
what programming language is the overlay using?

**Comment by piecedigital** [comment]
Just about everything is Python @FuzionDragonZKid

**Comment by faresfuad** [comment]
any progress guys?

**Comment by piecedigital** [comment]
@faresfuad It has been updated by someone already [here](https://github.com/WAZAAAAA0/TekkenBot/)

**Comment by FuzionDragonZKid** [comment]
I'm not sure what to do. I'm I suppose to update the "memory_address.config" file

**Comment by piecedigital** [comment]
No. He already did that and compiled the program. So you just need the latest release.

**Comment by FuzionDragonZKid** [comment]
I don't see it

**Comment by FuzionDragonZKid** [comment]
I see it my bad. thanks

**Comment by faresfuad** [comment]
thanks fam

**Comment by choi-choi** [comment]
seems to have been another update yesterday, overlay doesn't work for me.

same with you guys?

**Comment by FuzionDragonZKid** [comment]
same here doesn't work

**Comment by AlexLaRochelle** [comment]
Doesn't work for me either

**Comment by piecedigital** [comment]
The latest patch has changed the memory address.

[Copy the data from this file](https://github.com/WAZAAAAA0/TekkenBot/blob/master/TekkenData/memory_address.ini), replace the contents of `TekkenData/memory_address.ini`. It should work.
